content/en/docs/spinnaker-install-admin-guides/prometheus-monitoring.md

```diff
@@ -53,141 +62,190 @@ Access the Prometheus web interface by using the `kubectl port-forward` command.
 
 Navigate to `http://localhost:9090/targets`.
 
-## Configure monitoring in Spinnaker
+## Configure monitoring using the Observability Plugin 
 
-To enable monitoring of Spinnaker by Prometheus, enable the `metric-stores` configuration.
+{{% alert title="Caution" color=warning %}} Before configuring monitoring, read and understand the following information about the security implications.
+If any of your services, typically Gate, are exposed to the open internet, there is a
+risk that you can publicly expose information. Armory recommends that you filter
+these paths at your edge layer in some manner. Be aware of any endpoints you 
+expose. Spring boot exposes the health endpoint by default though with some
+restrictions on what information is exposed. When auth is enabled, Gate restricts 
+access to the endpoints other than `/health`, preventing access to metric data.
 
-**Halyard**
+There is an issue on the plugin to expose metrics without security allowing scraping, but this would
+also increase a risk of exposure. For more information, see [Armory Observability Plugin issue #20](https://github.com/armory-plugins/armory-observability-plugin/issues/20).
 
-Issue these halyard commands from within your hal directory or within your halyard container:
+For more information on Spring actuators, see the [Monitoring and Management](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-monitoring).  
 
-```bash
-halyard-0:~ $ hal config metric-stores prometheus enable
+<!-- Spinnaker issue discussing management endpoints: https://github.com/spinnaker/spinnaker/issues/3883--> 
```

Comment:
I commented out this line cause I wasn't sure what we wanted the end user to get out of it. Can add back in based on your input @jasonmcintosh